XenModel/Messages.resx

```diff
@@ -8461,7 +8461,7 @@ Once the VM has restarted click the Install [XenServer product] Tools menu item
     <value>&amp;New Policy...</value>
   </data>
   <data name="NEW_POOL" xml:space="preserve">

```

Comment:
I believe this string is not used anywhere any more - can we remove it to keep the code lean?